PendingReleaseNotes

```diff
@@ -0,0 +1,8 @@
+14.1.1
+------
+
+* The monitor cluster may now withhold filesystem and/or MDS information from
+  clients based on a new auth cap:
+  [mon] allow rw; allow fsid=1; allow fsid=2
+* Communication with MDS daemons may now be restricted using a new auth cap:
+  [mds] allow fsid=1
```

Comment:
Like `mds allow rw fsid=1`?

Reply:
This implementation doesn't have a read-only variant. If you're restricted by fsid, the mds will drop all messages for unauthorized fsids. Do we want a read-only variant?

Reply:
> This implementation doesn't have a read-only variant.

I think I was wondering if `mds allow r fsid=1` would be allowed. That is, read-only MDS access although I'm not sure who uses that in practice; would that imply a read-only mount?